docs/source/conf.py

```
@@ -192,6 +192,12 @@ def setup(app):
     generate_llmapi()
     update_version()
 
+    try:
+        from generate_transforms_doc import generate_transforms_reference
+        generate_transforms_reference()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)